wordpress-mobile/WordPress-iOS
# Add privacy declarations to the apps and extensions

See [this documentation](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api) regarding the privacy declaration requirement.

Here are the required declarations in our apps and extensions that are listed in App Store Connect email:

* PlugIns/JetpackDraftActionExtension.appex/JetpackDraftActionExtension
  - NSPrivacyAccessedAPICategoryFileTimestamp
  - NSPrivacyAccessedAPICategoryUserDefaults

* PlugIns/JetpackIntents.appex/JetpackIntents
  - NSPrivacyAccessedAPICategoryFileTimestamp
  - NSPrivacyAccessedAPICategoryUserDefaults

* PlugIns/JetpackNotificationServiceExtension.appex/JetpackNotificationServiceExtension
  - NSPrivacyAccessedAPICategoryUserDefaults

* PlugIns/JetpackShareExtension.appex/JetpackShareExtension
  - NSPrivacyAccessedAPICategoryFileTimestamp
  - NSPrivacyAccessedAPICategoryUserDefaults

* Jetpack
  - NSPrivacyAccessedAPICategorySystemBootTime
  - NSPrivacyAccessedAPICategoryDiskSpace
  - NSPrivacyAccessedAPICategoryFileTimestamp
  - NSPrivacyAccessedAPICategoryUserDefaults

* PlugIns/JetpackStatsWidgets.appex/JetpackStatsWidgets
  - NSPrivacyAccessedAPICategoryFileTimestamp
  - NSPrivacyAccessedAPICategoryUserDefaults


<details>
<summary>For easier review, here are the content of the privacy declaration files</summary>

| Bundle | PrivacyInfo |
| ------ | ----------- |
| App | <img width="873" alt="App" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/1101828/5a619912-0c8a-4d12-aac6-85606e9e8d78"> |
| Draft Action | <img width="863" alt="DraftAction" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/1101828/4f6a4a09-3ed2-4f1a-8722-38035d4772d2"> |
| Intent | <img width="863" alt="Intent" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/1101828/86942a68-9c6d-4d9a-b718-cb4afe6749da"> |
| Notification Service | <img width="864" alt="NotificationService" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/1101828/25d74c07-d02a-4529-bd84-917ca2d4a8eb"> |
| Share Extension | <img width="864" alt="ShareExtension" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/1101828/0f515f06-7920-4b54-b2b6-820e465392de"> |
| Stats Widget | <img width="870" alt="StatsWidget" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/1101828/aac9fe7d-8fa0-4201-8a61-7ad6d7f26624"> |

</details>


## Test Instructions

Unzip the ipa file from App Center, and ensure there is a "PrivacyInfo.xcprivacy" file for in the app and plugins directory listed above. Verify the content listed in the xcprivacy file matches the app or plugin it belongs to.


PR submission checklist:
